frontend/lib/norent/data/faqs-content.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+
+export type FaqCategory =
+  | "Letter Builder"
+  | "Tenant Rights"
+  | "Connecting With Others"
+  | "After Sending Your Letter";
+
+export type Faq = {
+  question: string;
+  category: FaqCategory;
+  priority: number;
+  answerPreview: React.ReactNode;
+  answerFull: React.ReactNode;
+};
+
+export const FaqsContent: Faq[] = [
+  {
```

Comment:
This can just be considered "dummy data" for now— I needed something to play with that fit the data model we're planning for.